content/en/docs/concepts/storage/volume-health-monitoring.md

```diff
@@ -0,0 +1,34 @@
+---
+reviewers:
+- jsafrane
+- saad-ali
+- msau42
+- xing-yang
+--
+title: Volume Health Monitoring 
+content_type: concept
+---
+
+<!-- overview -->
+
+This page describes the CSI Volume Health Monitoring feature. CSI Volume Health Monitoring allows CSI Drivers to detect abnormal volume conditions from the underlying storage systems and report them as events on PVCs or Pods.
+
+{{< feature-state for_k8s_version="v1.21" state="alpha" >}}
+
+<!-- body -->
+
+## Volume Health Monitoring
+
+The Volume Health Monitoring feature is part of Kubernetes implementation of Container Storage Interface (CSI). The Volume Health Monitoring feature is impelmeted in two components: External Health Monitor Controller and Kubelet.
+
+The External Health Monitor Controller will be deployed as a sidecar together with the CSI controller driver. It triggers controller RPC to check the health condition of the CSI volumes.
+
+The external controller sidecar will also watch for node failure events. This can be enabled by setting the `enable-node-watcher` flag to true. This will only have effects on local PVs now. When a node failure event is detected, an event will be reported on the PVC to indicate that pods using this PVC are on a failed node.
```

Comment:
By  "local PV", do you mean the in-tree "kubernetes.io/local" volume type, or just in general, local volume types?

Reply:
This does not refer to the in-tree "kubernetes.io/local" volume type because the volume health feature is only available to CSI drivers.  So this refers to local volume type which has a CSI driver.  Let me clarify that in the doc.